orange/src/test/java/com/hartwig/hmftools/orange/conversion/TestCommonDatamodelFactory.java

```diff
@@ -0,0 +1,39 @@
+package com.hartwig.hmftools.orange.conversion;
+
+import java.util.List;
+
+import com.hartwig.hmftools.common.virus.AnnotatedVirus;
+import com.hartwig.hmftools.common.virus.ImmutableAnnotatedVirus;
+import com.hartwig.hmftools.common.virus.ImmutableVirusInterpreterData;
+import com.hartwig.hmftools.common.virus.VirusBreakendQCStatus;
+import com.hartwig.hmftools.common.virus.VirusInterpreterData;
+import com.hartwig.hmftools.common.virus.VirusLikelihoodType;
+
+class TestCommonDatamodelFactory {
+
+    static VirusInterpreterData emptyVirusInterpreterData() {
+        return ImmutableVirusInterpreterData.builder().build();
+    }
+
+    static VirusInterpreterData minimalVirusInterpreterData() {
+        AnnotatedVirus virus = minimalAnnotatedVirusBuilder().build();
+        return ImmutableVirusInterpreterData.builder().addReportableViruses(virus).addAllAllViruses(List.of(virus)).build();
```

Comment:
Do you need `addAllAllViruses()` here or could you just provide `virus` to `addAllViruses()` without having to build a list?